shared/chat/conversation/input-area/suggestors/index.stories.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+import * as React from 'react'
+import * as Sb from '../../../../stories/storybook'
+import * as Kb from '../../../../common-adapters'
+import * as Styles from '../../../../styles'
+import AddSuggestors, * as Suggestors from '.'
+
+class _TestArea extends React.Component<Suggestors.PropsWithSuggestor<{somethingElse: 'this'}>> {
+  render() {
+    return (
+      <Kb.Box2 direction="vertical" gap="tiny" style={{padding: 10}}>
+        <Kb.Text type="BodySmall" selectable={true}>
+          Available triggers: {availableTriggers.toLocaleString()}
+        </Kb.Text>
+        <Kb.PlainInput
+          onBlur={this.props.onBlur}
+          onFocus={this.props.onFocus}
+          onChangeText={this.props.onChangeText}
+          onKeyDown={this.props.onKeyDown}
+          onSelectionChange={this.props.onSelectionChange}
+          multiline={true}
+          rowsMax={3}
+          ref={this.props.inputRef}
+          style={{
+            borderColor: 'black',
+            borderStyle: 'solid',
+            borderWidth: 1,
+            width: 200,
+          }}
+        />
+      </Kb.Box2>
+    )
+  }
+}
+const TestArea = AddSuggestors(_TestArea)
+
+// prettier-ignore
+const fruit = ['apple', 'orange', 'raspberry', 'cantaloupe', 'durian', 'blackberry', 'fruit (generic)', 'mango', 'nectarine', 'pineapple', 'lemon']
```

Comment:
If I type "$" and select "fruit (generic)" and then something else, only the "(generic)" part is removed, leaving $fruit and another "$" to be autocompleted. Not sure if this is just a story bug or a real bug.

Reply:
It's a story bug, it's because "fruit (generic)" has a space in it, and the `getWordAtCursor` function assumes suggestions can't have spaces.